src/valkey-benchmark.c

```diff
@@ -2352,6 +2358,19 @@ int main(int argc, char **argv) {
             free(cmd);
         }
 
+        if (test_is_selected("zscore")) {
```

Comment:
This test is after the zpopmin test which is after the zadd test. I guess the zpopmin test deletes all the elements added by the zadd.

Let's move these new tests before the zpopmin test?

Reply:
Good point. I did not consider the fact that the tests will "just be run" :)

Reply:
You've tried it locally, haven't you? ;)